bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-100to200/ZJetsToNuNu_HT100to200_proc_card.dat

```diff
@@ -0,0 +1,41 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-ckm_no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+# Define multiparticle labels
+define p u c s d b u~ c~ s~ d~ b~ g
+define j  = p
+# Specify process(es) to run
+generate p p > ve ve~ / h @0
+add process p p > ve ve~ j / h @1
```

Comment:
I think this is a wrong convention adopted long time ago. If one only uses the ve the cross section will be wrong by a factor 3, so I suggest to use vl defined above

Reply:
Hi,
I will make the changes on the ZJets cards, meanwhile could you please check the latest commit for the EWK and WG? Thanks.